nix/cells/cloud/actions.nix

```diff
@@ -21,4 +19,26 @@
       ]
     '';
   };
+
+  "plutus/publish-unraveling-recursion-paper" = {
+    task = "publish-unraveling-recursion-paper";
+    io = ''
+      let push = {
+        #lib.io.github_push
+        #input: "${cell.library.actions.paper.input}"
+        #repo: "input-output-hk/plutus"
+        inputs: _final_inputs
+      }
+
+      _final_inputs: inputs
```

Comment:
this all looks weird but I assume you know what you're doing. Maybe could do with some comments :)